yarn.lock

```diff
@@ -4705,6 +4712,14 @@ tapable@^0.2.3, tapable@~0.2.3:
   version "0.2.4"
   resolved "https://registry.yarnpkg.com/tapable/-/tapable-0.2.4.tgz#a7814605089d4ba896c33c7e3566e13dcd194aa5"
 
+tar-fs:
```

Comment:
Could you use latest master branch to run `yarn add tar-fs` for this change?
In 0.16 we had an issue that generated a lockfile entry without version